__bass.py

```diff
@@ -24,11 +24,13 @@ def gen_script():
                        .check_output(['bash', '-c', command])
                        .split(divider, 1))
     new_env = new_env.lstrip().splitlines()
-
+    
+    new_env = [line for line in new_env if '{' not in line and '}' not in line]
```

Comment:
With quoting add by you on line 52, there is no need to exclude lines with `{` or `}`.